### PR TITLE
cargo-make: init at 0.17.1

### PR DESCRIPTION
**Pull request by xrelkd**
###### Motivation for this change

`cargo-make` is a Rust runner and build tool.

[https://sagiegurari.github.io/cargo-make/](https://sagiegurari.github.io/cargo-make/)
[https://github.com/sagiegurari/cargo-make](https://github.com/sagiegurari/cargo-make)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by xrelkd** [comment]
@GrahamcOfBorg build cargo-make

File: pkgs/development/tools/rust/cargo-make/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  cargoSha256 = "18jmk6fk59qvxb34ia8vxww09p616a2c3nxs981sl65p84fvlgrp";
+  cargoPatches = [ ./fix-build.patch ];
+
+  # test result: FAILED. 589 passed; 33 failed; 0 ignored; 0 measured; 0 filtered out
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "A Rust task runner and build tool";
+    homepage = https://github.com/sagiegurari/cargo-make;
+    license = with licenses; [ asl20 ];
```
Comment (pSub) [inline pkgs/development/tools/rust/cargo-make/default.nix]: `license` is not a list.

File: pkgs/development/tools/rust/cargo-make/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    repo = pname;
+    rev = version;
+    sha256 = "1hsncyidzh41fmrb29f8n7k5ij28wh8b74j8wa1ikimsvzpqya6p";
+  };
+
+  cargoSha256 = "18jmk6fk59qvxb34ia8vxww09p616a2c3nxs981sl65p84fvlgrp";
+  cargoPatches = [ ./fix-build.patch ];
+
+  # test result: FAILED. 589 passed; 33 failed; 0 ignored; 0 measured; 0 filtered out
+  doCheck = false;
```
Comment (pSub) [inline pkgs/development/tools/rust/cargo-make/default.nix]: Is there something you could do to make the failing tests succeed?

**Review by pSub — CHANGES_REQUESTED** [review]

**Comment by xrelkd** [comment]
@pSub Thanks for the review!
Some tests fail because they need network access.
However, Travis ensures a proper build.
See also: https://travis-ci.org/sagiegurari/cargo-make/builds/515874289

**Comment by pSub** [comment]
@GrahamcOfBorg build cargo-make

**Comment by xrelkd** [comment]
Hello @pSub, Thanks for the review.
I guess this can be merged now, thank you!

**Pull request closed without merging**